packages/devtools/lib/src/memory/memory_protocol.dart

```diff
@@ -140,3 +140,73 @@ class HeapSample {
   final int external;
   final bool isGC;
 }
+
+// Heap Statistics
+
+// {
+//   type: ClassHeapStats,
+//   class: {type: @Class, fixedId: true, id: classes/5, name: Class},
+//   new: [0, 0, 0, 0, 0, 0, 0, 0],
+//   old: [3892, 809536, 3892, 809536, 0, 0, 0, 0],
+//   promotedInstances: 0,
+//   promotedBytes: 0
+// }
+class ClassHeapStats {
```

Comment:
nit: maybe choose a different name to avoid a conflict with the VMService class with the same name?
Also, consider adding a comment indicating why we aren't using the VMService class.

Reply:
it might make sense to upstream this change to the VMService package instead although I don't have context to know whether that is a good idea.

Reply:
Done